test/distributed/test_c10d.py

```diff
@@ -81,6 +82,14 @@ def gpus_for_rank(world_size):
         )
     return gpus_for_rank
 
+def create_client(i, addr, port, world_size=-1, timeout=timedelta(seconds=10)):
+    if world_size < 0:
```

Comment:
nitpick: Do you need a conditional here? Passing `world_size=-1` should behave anyways as if it was not provided.